### DISCUSSION TITLE
Does custom tkinter have notebook like ttk.Notebook()

### DISCUSSION THREAD

**Discussion by babymu5k** [TomSchimansky/CustomTkinter#1303] [Q&A] [answered]
Does custom tkinter have notebook like ttk.Notebook()

**Comment by ElectricCandlelight** [comment, marked as answer]
You can see a picture of all the widgets together on the main page.

CTkTabview 

[https://github.com/TomSchimansky/CustomTkinter/wiki/CTkTabview](https://github.com/TomSchimansky/CustomTkinter/wiki/CTkTabview)

**Reply by babymu5k** [reply]
Thanks 👍
